examples/reference/panes/HoloViews.ipynb

```diff
@@ -119,18 +207,205 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "widgets = hv_panel[1]\n",
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\")\n",
+    "pn.pane.HoloViews(plot, backend='matplotlib', sizing_mode=\"stretch_width\", height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please note that in a *server context* you might have to set the matplotlib backend like below depending on your setup.\n",
+    "\n",
+    "```python\n",
+    "import matplotlib\n",
```

Comment:
Is the comment below still needed?

```python
import matplotlib
matplotlib.use('agg')
```

I did never do that. And it still works.